.github/workflows/tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     strategy:
       matrix:
         wp-versions: [ 'latest' ] #[ 'latest', '6.1.1' ]
-        php-versions: [ '7.4', '8.0', '8.1', '8.2' ] #[ '7.4', '8.0', '8.1', '8.2' ]
+        php-versions: [ '7.4', '8.0', '8.1', '8.2', '8.3' ] #[ '7.4', '8.0', '8.1', '8.2' ]
```

Comment:
Why are we deleting the ngnix config for 7.4, 8.0, and 8.1 if we're still testing against them?

Reply:
nginx is never used in the GitHub action for running unit tests, so the nginx config files aren't needed here.

Reply:
Got it, what are they for then?

Reply:
The `php-versions` are used in the testing matrix to run unit tests against each PHP version (example results can be seen [here](https://github.com/ConvertKit/convertkit-wordpress-libraries/actions/runs/7103955903)).

nginx configuration files remain in the Plugin repositories, as those GitHub actions run acceptance tests through the browser, interacting with WordPress and a Plugin (e.g. adding a ConvertKit Form block). nginx acts as the web server for running WordPress for those tests.